CMakeLists.txt

```diff
@@ -986,12 +989,14 @@ elseif(${PLAYER_TARGET_PLATFORM} MATCHES "^(3ds|switch|psvita)$")
 			FILE "${CMAKE_CURRENT_SOURCE_DIR}/resources/psvita/touch_ui.png" touch_ui.png)
 	endif()
 	# for distribution
-	add_custom_command(TARGET easyrpg-player POST_BUILD
-		BYPRODUCTS easyrpg-player.elf
-		COMMAND ${CMAKE_COMMAND} -E copy_if_different
-			easyrpg-player easyrpg-player.elf
-		WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-		COMMENT "Prepare for distribution")
+	if(EXISTS "${CMAKE_CURRENT_BINARY_DIR}/easyrpg-player")
```

Comment:
When using DKP toolchain the file is resulting file is already called "easyrpg-player.elf" so I'm not sure if we even need this custom command anymore. Can also remove it.

Reply:
That would be okay. I added it just because we have shipped the elf ever since and it would otherwise require a `cp` on jenkins. But since it is only used now by vita, we could also likely set `CMAKE_EXECUTABLE_SUFFIX=.elf` there and remove this block.

Reply:
You only get elf when you have recent cmake, btw: https://github.com/devkitPro/pacman-packages/blob/7a3febdaa5e784bce61bbbdcad94d9eb65e828f2/cmake/common-utils/dkp-toolchain-common.cmake#L9-L11